### ISSUE TITLE
Sum label inside a donut chart

### ISSUE THREAD

**Issue by ionstudio** [amcharts/amcharts4#3527]
On the page https://www.amcharts.com/docs/v4/tutorials/sum-label-inside-a-donut-chart/ JSON example does not work!

https://codepen.io/team/amcharts/pen/wZLwyW?editors=0010

How to display the text of the sum in the center of the circle using JSON?

**Comment by martynasma** [comment]
Thanks for the heads up!

We just updated the CodePen and the tutorial.

**Comment by andykno** [comment]
The updated tutorial does not work if a legend is added to the chart.

![image](https://user-images.githubusercontent.com/15866970/121701292-0a04e280-cad1-11eb-8876-70be9f5a5f5c.png)

The old tutorial/CodePen workted until version 4.10.17.

**Comment by martynasma** [comment]
You may try adjusting label using `dx`:

```JSON
    "bulletsContainer": {
      "children": [{
        "type": "Label",
        "forceCreate": true,
        "text": "${values.value.sum}",
        "horizontalCenter": "middle",
        "verticalCenter": "middle",
        "fontSize": 40,
        "dx": -120
      }]
```

**Comment by andykno** [comment]
Thanks for your response.

Using `dx` or `dy` does not work.

The position of the label is wrong if you change the window size after the chart has been rendered.

Moreover it is to difficult to calculate the value of `dx` or `dy`.
For example: If you change the position of the legend to bottom, the value of `dy` depends on the number of rows of the legend.

**Comment by martynasma** [comment]
In that case it's probably better to put label in different container as well as use events to assign label's data item:

https://codepen.io/team/amcharts/pen/5db9a20f278409427f714e9ddd285db1?editors=0010

**Comment by mohd-akram** [comment]
Looks like this was fixed in 4.10.23.
